docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+  pgduckdb:
+    container_name: pgduckdb
+    image: pgduckdb/pgduckdb:16-main
```

Comment:
Something is wrong with how we publish images. This tag is only available for arm64, while some others are available for amd64. But non are available for both.

Reply:
opened #307